userspace/otpilot/Build.mk

```diff
@@ -15,3 +15,12 @@
 # SPDX-License-Identifier: Apache-2.0
 
 RUST_APPS_papa += otpilot
+
+# .PHONY so that it happens every time
+.PHONY: build/gitlongtag
+
+build/gitlongtag:
+	mkdir -p build
+	git describe --always --dirty --long >build/gitlongtag
+
+sandbox_setup: build/gitlongtag
```

Comment:
I think it would be better to make `build/gitlongtag` a dependency of the targets that call `cargo check`, this isn't related to sandbox setup. I.e., starting at [userspace/Build.mk line 261](https://github.com/google/tock-on-titan/blob/2e3f84edf8d1f97f6da4a6bd0a3eedebde46db81/userspace/Build.mk#L261), add it as a dependency to `check`, `doc`, and `app`.

Reply:
Yes, new push puts it all in `userspace/Build.mk` with the dependencies you suggested.   Seems to work correctly with those dependencies.